src/script/services/manifest.ts

```diff
@@ -99,33 +97,35 @@ export async function fetchManifest(url: string): Promise<ManifestDetectionResul
   // 2. An Azure function that uses Chrome Puppeteer to fetch the manifest
```

Comment:
Comment seems out of date (step 1 was removed, IIRC)

Reply:
removed in this PR https://github.com/pwa-builder/PWABuilder/pull/1535